boilerplate/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "cloud_based_e-commerce_platform",
+  "version": "1.0.0",
+  "description": "A cloud based E-commerce platform",
+  "author": "Darshit <darshit.chanpura@gmail.com>, Samarth Parikh, Alex Balgavy, Karthik Mynam",
+  "license": "ISC",
+  "repository": "https://github.com/BU-NU-CLOUD-F19/Cloud_Based_E-Commerce_Platform",
+  "homepage": "https://github.com/BU-NU-CLOUD-F19/Cloud_Based_E-Commerce_Platform",
+  "bugs": "",
+  "config": {
+    "appName": "api"
+  },
+  "scripts": {
+    "dev": "nodemon",
+    "start": "node app.js",
+    "lint": "eslint . --fix"
+  },
+  "dependencies": {
+    "bluebird": "3.5.5",
+    "chai": "^4.0.2",
+    "elv": "^1.0.1",
+    "express": "4.16.4",
+    "@hapi/glue": "6.2.0",
+    "@hapi/hapi": "18.4.0",
+    "hapi-swagger": "^7.0.0",
+    "jerkface": "^1.0.0",
+    "kibbutz": "^1.2.0",
+    "kibbutz-rc": "^1.0.0",
+    "knex": "^0.14.2",
+    "lodash": "^4.17.4",
+    "moment": "^2.18.1",
+    "npx": "^10.2.0",
+    "pg": "^7.4.1",
+    "qs": "^6.5.1",
+    "uuid": "^3.0.1",
+    "winston": "3.2.1",
+    "winston-daily-rotate-file": "4.1.0"
+  },
+  "devDependencies": {
+    "eslint": "^3.13.1",
```

Comment:
Please use the newest version whenever you can (and not of just eslint, of all packages), this is more than 3 major versions behind.